SU2_CFD/src/interfaces/fsi/CDisplacementsInterface.cpp

```diff
@@ -44,7 +44,7 @@ void CDisplacementsInterface::GetDonor_Variable(CSolver *struct_solution, CGeome
   for (auto iVar = 0u; iVar < nVar; iVar++)
     Donor_Variable[iVar] = DisplacementDonor[iVar];
 
-  if (struct_config->GetTime_Domain()) {
+  if (struct_config->GetTime_Domain() && !struct_config->GetWrt_Slice()) {
```

Comment:
I clicked (kindof) randomly on this commit and saw this `WRT_SLICE` option which was just deprecated in #1129 ... oh and I just also just saw that you hooked it because it is otherwise unused :D I already wondered why it would mater here ... So I guess you have to go for another one if you merge develop, sorry 

I am guilty of doing that as well ;)

Reply:
That's no problem at all. I just wanted to test the new velocity interface without having to recompile every time when I didn't know whether this interface would be adopted or not in the end. But this has been validated now and so I'll set up proper options when I move forward with this PR...